README.md

```diff
@@ -67,4 +79,4 @@ then running your tests: `cargo test && cargo insta review`
 
 if the info log isn't enough, run `DEBUG=true cargo test`.
 if you wish to pipe to a file run `NO_COLOR=true cargo test`
-you may also set the `NOLOG` env variable if you wish to disable logging entirely.
\ No newline at end of file
+you may also set the `NOLOG` env variable if you wish to disable logging entirely.
```

Comment:
has something change here?

Reply:
Yes, my editor insert a EOL in the file.